overlord/snapstate/snapmgr.go

```diff
@@ -257,7 +259,12 @@ func readInfo(name string, si *snap.SideInfo, flags int) (*snap.Info, error) {
 		if si != nil {
 			info.SideInfo = *si
 		}
-		return info, nil
+		err = nil
+	}
+	if err == nil && flags&loadStore != 0 {
+		if e := backend.AttachStoreInfo(info); e != nil {
+			logger.Debugf("cannot read cached snap store info for %q: %v", name, e)
```

Comment:
```suggestion
			logger.Debugf("cannot attach cached snap store info for %q: %v", name, e)
```

Reply:
I think read is fine, attach is really an internal concept here, not sure why expose it